feature_aggregation_scripts/ocean_subBasin_regions/ocean_subBasin_regions.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python
+
+import shapely
+import numpy
+import xarray
+import os
+import matplotlib.pyplot as plt
+import pyproj
+import gzip
+import shutil
+import tempfile
+
+from geometric_features import GeometricFeatures, FeatureCollection
+from geometric_features.feature_collection import _round_coords
+
+from geometric_features.download import download_files
+from geometric_features.utils import write_feature_names_and_tags
+
+
+def main():
```

Comment:
Please rename this and give it a docstring in numpydoc format:
```suggestion
def subbasins(gf):
    """
    Aggregates ocean regions into the following larger sub-basins:
    * Arctic Ocean Basin
    * North Atlantic Basin
    * South Atlantic Basin
    * North Pacific Basin
    * South Pacific Basin
    * Indian Ocean Basin
    * Southern Ocean Basin
    
    Parameters
    ----------
        gf : ``GeometricFeatures``
        An object that knows how to download and read geometric featuers

    Returns
    -------
    fc : geometric_features.FeatureCollection
        The new feature collection with ocean sub-basins
```